src/app/content/search/components/SearchResultsSidebar/SearchResultHits.tsx

```diff
@@ -83,4 +89,11 @@ const SearchResultHits = ({
   </React.Fragment>;
 };
 
-export default SearchResultHits;
+// tslint:disable-next-line:variable-name
+export const ConnectedSearchResultHits = connect(
+  (state: AppState) => ({
+    queryParams: navSelect.persistentQueryParameters(state),
+  })
+)(SearchResultHits);
```

Comment:
@jivey connecting this component to the store seems to have fixed the search query bug

Reply:
(thanks for finding that bug btw)

Reply:
Ah nice!